containers/monitoring-alertmanager-image/Dockerfile

```
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: MIT
+
+#     Copyright (c) 2024 SUSE LLC
+
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon.
+
+#!BuildTag: uyuni/alertmanager:%%alertmanager_patch_version%%
+#!BuildTag: uyuni/alertmanager:latest
```

I made  similar comments also later. If we publish via MLM we should align.

Plus,  I think  you don't need the additional  patch_version tag because we add the rest when we  push  to OBS/IBS. But better to check this part with Hexagon.

```suggestion
#!BuildTag: uyuni/server-monitoring-alertmanager:latest
```